### ISSUE TITLE
[Feature request] Tutorial

### ISSUE THREAD

**Issue by EmmanuelMess** [arpitkh96/AmazeFileManager#646]
There seems to be that there are a lot of features in this app that go under the radar of some users, such as the multiple tabs, shortcutting, long tapping storages in the drawer, etc.. It would be a good idea to showcase these in a simple tutorial.

**Comment by vishal0071** [comment]
Indeed.

**Comment by EmmanuelMess** [comment]
Related: [material guidelines' gesture education](https://material.io/guidelines/growth-communications/gesture-education.html).

**Comment by Primokorn** [comment]
Multiple tabs? How ?

**Comment by EmmanuelMess** [comment]
@Primokorn In other OSs you might have the option of opening a new window in the explorer, here you can swipe left or right.

**Comment by Primokorn** [comment]
@EmmanuelMess Yeah I though you were talking about more than 2 tabs/panels (not enough on my end).

**Comment by VishnuSanal** [comment]
tutorials can be added to discussions, if someone is working on this, start a "show and tell" discussion.
